modules/patterns/pages/github-merge-queues.adoc

```diff
@@ -0,0 +1,64 @@
+= GitHub Merge Queues
+
+This document contains instructions for how to configure configure GitHub link:https://docs.github.com/en/repositories/configuring-branches-and-merges-in-your-repository/configuring-pull-request-merges/managing-a-merge-queue[merge queues] with {ProductName}.
```

Comment:
```suggestion
This document contains instructions for how to configure GitHub link:https://docs.github.com/en/repositories/configuring-branches-and-merges-in-your-repository/configuring-pull-request-merges/managing-a-merge-queue[merge queues] with {ProductName}.
```

Reply:
Somehow we're back with `configure` appearing twice :)

Reply:
otherwise lgtm

Reply:
Some force-push error on my part. :(